src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -815,6 +815,54 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
         return builder->CreateCall(fn, {str, idx1, idx2});
     }
 
+    llvm::Value* lfortran_float_to_str(llvm::Value* number)
+    {
+        std::string func_name = "_lfortran_float_to_str";
+        llvm::Function *fn = module->getFunction(func_name);
+        if (!fn) {
+            llvm::FunctionType *function_type = llvm::FunctionType::get(
+                    character_type, {
+                        llvm::Type::getFloatTy(context)
```

Comment:
Use `getFPType` defined in `asr_to_llvm.cpp`. Will this code if I pass a 64 bit floating point here?